contracts/oracles/chainlink/ChainlinkUSDMultiOracle.sol

```diff
@@ -104,3 +105,52 @@ contract ChainlinkUSDMultiOracle is IOracle, AccessControl, Constants {
         updateTime = (updateTime1 < updateTime2) ? updateTime1 : updateTime2;
     }
 }
+
+/**
+ * @title ChainlinkL2USDMultiOracle: ChainlinkUSDMultiOracle that's safe to use on L2
+ * @notice Chainlink recommends checking the sequencer status on some L2 networks to avoid
+  reading stale data
+
+  https://docs.chain.link/docs/l2-sequencer-flag/
+ */
+contract ChainlinkL2USDMultiOracle is ChainlinkUSDMultiOracle {
```

Comment:
Didn't notice this before, could you please split ChainlinkL2USDMultiOracle into its own file?